src/bootloader.c

```diff
@@ -153,6 +155,90 @@ void LuosBootloader_SaveLastData(void)
     LuosHAL_ProgramFlash(page_addr, page_id, (uint16_t)PAGE_SIZE, page_buff);
 }
 
+/******************************************************************************
+ * @brief compute crc 8 for each data
+ * @param data pointer, data len 
+ * @return crc
+ ******************************************************************************/
+void crc8(const uint8_t *data, uint8_t *crc, uint16_t polynome)
```

Comment:
@nicolas-rabault should we keep 2 crc calculation different? there is no time calculation gain with HW CRC on 1 byte in luos communication

Reply:
I don't get it... Here this is a CRC used to check the binary.
I don't see 2 different CRC calculations.